containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -178,6 +179,46 @@ public void addServletHandlers(String serverHostName) {
             rewriteHandler.addRule(disableKeepAliveRule);
         }
         
+        // Security: Respond the "Expect-CT: max-age=31536000, report-uri=..." header
+
+        HeaderPatternRule expectCtRule = new HeaderPatternRule();
+        expectCtRule.setPattern("/*");
+        expectCtRule.setName("Expect-CT");
+        expectCtRule.setValue("max-age=31536000, report-uri=\"http://csp.yahoo.com/beacon/csp?src=yahoocom-expect-ct-report-only\"");
+        rewriteHandler.addRule(expectCtRule);
+
+        // Security: Respond the "Strict-Transport-Security: max-age=31536000" header
+
+        HeaderPatternRule hstsRule = new HeaderPatternRule();
+        hstsRule.setPattern("/*");
+        hstsRule.setName(HttpHeader.STRICT_TRANSPORT_SECURITY.asString());
+        hstsRule.setValue("max-age=31536000");
```

Comment:
let's make the age as configurable as with a default value.

Reply:
Right